### PR TITLE
Translate Abstract Factory to Russian / Fix #2286 partially

### LINKED ISSUE #2286: Translation to Russian
This task tracks the translation of the patterns. More information about the translation process is available at https://github.com/iluwatar/java-design-patterns/wiki/15.-Support-for-multiple-languages

Acceptance criteria

- The patterns have been translated
- The patterns can be browsed in Github and on the website https://java-design-patterns.com

### PR DESCRIPTION
**Pull request by VladimirCreator**
This pull request localizes the description of the abstract factory design pattern to Russian.

### CODE REVIEW

**Comment by VladimirCreator** [comment]
You can assign the issue #2286 to me.

File: localization/ru/abstract-factory/README.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
+---
+title: Abstract Factory
+category: Creationsl
```
Comment (iluwatar) [inline localization/ru/abstract-factory/README.md:3]: should be Creational

**Review by iluwatar — CHANGES_REQUESTED** [review]

**Comment by iluwatar** [comment]
> You can assign the issue #2286 to me.

Please comment on the issue, otherwise I am not able to assign it to you

**Review by iluwatar — APPROVED** [review]

**Comment by iluwatar** [comment]
Looks good! Thank you for the contribution 🎉

@all-contributors please add @VladimirCreator for translation

**Pull request merged**